ovirt_imageio/_internal/handlers/tickets.py

```diff
@@ -21,9 +21,19 @@ class Handler:
     Handle requests for the /tickets/ resource.
     """
 
-    def __init__(self, config, auth):
+    def __init__(self, config, auth, ticket=None):
```

Comment:
A handler is never created with a ticket. We don't want to add such argument and have inconsistent signature.

We can handle this in the server module - accept a --ticket argument, and add the ticket to the autorizer. The request handler does not need to know anything about initial ticket.

All the changes in this module are not needed.